### PR TITLE
booking confirmation

### PR DESCRIPTION
**Pull request by Actionmaker1**
functional booking confirm page and booked spaces overview

### CODE REVIEW

File: app/assets/stylesheets/components/_index.scss [hunk 1]
```diff
@@ -2,3 +2,7 @@
 @import "alert";
 @import "avatar";
 @import "navbar";
+.card-none {
+  text-decoration: none; 
+  color: #fca311;
+}
```
Comment (dwiehoff) [inline app/assets/stylesheets/components/_index.scss]: No difference, can be removed.

File: app/models/reservation.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class Reservation < ApplicationRecord
   belongs_to :space
   belongs_to :user
+  has_one_attached :image
```
Comment (dwiehoff) [inline app/models/reservation.rb]: Should come from space rather than reservation itself; likely unnecessary?

File: config/routes.rb [hunk 3]
```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   resources :spaces do
-    resources :reservations, only: [:new, :create]
+    resources :reservations, only: [:new, :create, :index, :show]
   end
+  get "reservations", to: 'reservations#index'
```
Comment (dwiehoff) [inline config/routes.rb:5]: duplicate

**Review by dwiehoff — CHANGES_REQUESTED** [review]

File: app/models/reservation.rb (hunk 2 above)
Comment (Actionmaker1) [reply app/models/reservation.rb]: true, was early try. ill check

File: config/routes.rb (hunk 3 above)
Comment (Actionmaker1) [reply config/routes.rb:5]: TA said I have to write it outside as well. He dictated it haha. But to be honest that guy had no idea.

Comment (dwiehoff) [reply config/routes.rb:5]: ah sorry makes sense.. keep it that way 👍

**Pull request merged**